cvs -f log -N -r1.12 kgvconfigdialog.cpp

RCS file: /home/kde/kdegraphics/kghostview/kgvconfigdialog.cpp,v
Working file: kgvconfigdialog.cpp
head: 1.12
branch:
locks: strict
access list:
keyword substitution: kv
total revisions: 12;	selected revisions: 1
description:
----------------------------
revision 1.12
date: 2002/08/27 12:43:55;  author: mlaurent;  state: Exp;  lines: +39 -47
Fix configure gs :
We must compare string to string and not version as a double
because now gs version is 7.05.4 => not a double
=============================================================================
cvs -f diff -bp -u -r1.11 -r1.12 kgvconfigdialog.cpp
Index: kgvconfigdialog.cpp
===================================================================
RCS file: /home/kde/kdegraphics/kghostview/kgvconfigdialog.cpp,v
retrieving revision 1.11
retrieving revision 1.12
diff -b -p -u -r1.11 -r1.12
--- kdegraphics/kghostview/kgvconfigdialog.cpp	2002/08/14 09:41:33	1.11
+++ kdegraphics/kghostview/kgvconfigdialog.cpp	2002/08/27 12:43:55	1.12
@@ -117,7 +117,6 @@ bool KGVConfigDialog::slotConfigureGhost
     if( !KStandardDirs::findExe( exe ) ) {
 	return false;
     }
-  
     QString versionString; 
     QString chkVersion = exe + " --version";
     FILE* p = popen( chkVersion.local8Bit(), "r" );
@@ -133,16 +132,9 @@ bool KGVConfigDialog::slotConfigureGhost
 
     versionString = versionString.stripWhiteSpace();
     
-    bool isNumber;
-    float versionNum = versionString.toFloat( &isNumber );
-    if( !isNumber ) {
-	// This shouldn't happen, so no need for a messagebox.
-	return false;
-    }
-
     mInterpreterPath = exe;
     mNonAntiAliasArgs = "-sDEVICE=x11";
-    mAntiAliasArgs =  versionNum < 6.50 
+    mAntiAliasArgs =  versionString < QString::number(6.50)
 		? "-sDEVICE=x11alpha"
 		: "-sDEVICE=x11 -dTextAlphaBits=4 -dGraphicsAlphaBits=2 " 
 		  "-dMaxBitmap=10000000";
